src/Stryker.Core/Stryker.Core/Compiling/CompilingProcess.cs

```diff
@@ -11,24 +11,10 @@
 
 namespace Stryker.Core.Compiling
 {
-    public interface ICompilingProcess
-    {
-        /// <summary>
```

Comment:
If the interface is not needed it can be removed, but the docstring in this case is very helpful in explaining what the parameters do, as that's not clear to someone new to the project